src/ser.rs

```diff
@@ -123,11 +123,7 @@ fn maybe_display_param(writer: &mut impl fmt::Write, key: impl fmt::Display, val
 #[rustfmt::skip]
 impl<'a, T> fmt::Display for Uri<'a, bitcoin::address::NetworkChecked, T> where for<'b> &'b T: SerializeParams {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        if f.alternate() {
```

Comment:
This was confusing. I could not find anywhere a `fn alternate` function.
I also looked into some deps but with no avail.
If there is a more elegant solution, please point me towards that.

Reply:
https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.alternate

@Kixunil  may we still want `bitcoin:{:#}` if `f.alternate()`?